src/ucp/core/ucp_request.inl

```diff
@@ -155,10 +157,37 @@
         return UCS_STATUS_PTR(UCS_ERR_INVALID_PARAM); \
     }
 
+static UCS_F_ALWAYS_INLINE void
+ucp_request_debug_string_null(ucp_request_t *req)
+{
+    #if ENABLE_DEBUG_DATA
+        req->debug_string = NULL;
+    #endif
+}
+
+static UCS_F_ALWAYS_INLINE void
+ucp_request_debug_string_init(ucp_request_t *req, const ucp_request_param_t *param)
+{
+    #if ENABLE_DEBUG_DATA
+        if (ucs_unlikely(param->op_attr_mask & UCP_OP_ATTR_FIELD_DEBUG_INFO)) {
+            req->debug_string =
```

Comment:
check for `NULL`

Reply:
`UCS_STRING_BUFFER_FIXED` allocates string buffer on stack. Request is a long lived object. Need string buffer to be a valid field

Reply:
UCS_STRING_BUFFER_FIXED is not necessarily on the stack, it can be any backing storage buffer